odoo/_monkeypatches/__init__.py

```diff
@@ -74,7 +74,7 @@ def patch_module(name):
     patch_module('codecs')
     patch_module('win32')
     for name in (
-        'ast', 'email', 'mimetypes', 'num2words', 'pytz', 'stdnum', 'urllib3',
-        'werkzeug', 'zeep',
+        'ast', 'csv', 'email', 'mimetypes', 'num2words', 'PIL', 'pytz', 're', 'stdnum', 'urllib3',
```

Comment:
modules should be lowercase
```suggestion
        'ast', 'csv', 'email', 'mimetypes', 'num2words', 'pil', 'pytz', 're', 'stdnum', 'urllib3',
```
also nitpicking: pep8 says 79 chars, and it is very easy to respect that here (here on github `'urllib3'` got moved to the next line, because it lacks space)

Reply:
PIL does not pollow the PEP :(